apps/storybook-react-native/stories/WalletHome.stories.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import type { Meta, StoryObj } from '@storybook/react-native';
+import { ScrollView, View, Text as RNText } from 'react-native';
+import { useTailwind } from '@metamask/design-system-twrnc-preset';
+
+/**
+ * Using DSRN components in this story breaks storybook see issue  https://github.com/MetaMask/metamask-design-system/issues/652
+ */
+import { Text } from '@metamask/design-system-react-native';
```

Comment:
Currently this `Text` component is not being used and storybook is rendering this story.